src/procedures/ModifyShareholderData.ts

```diff
@@ -17,12 +17,23 @@ import { Shareholder, SecurityToken } from '../entities';
 
 const { dateToBigNumber } = conversionUtils;
 
+/**
+ * Procedure that modifies data of a list of investors
+ */
 export class ModifyShareholderData extends Procedure<
   ModifyShareholderDataProcedureArgs,
   Shareholder[]
 > {
   public type = ProcedureType.ModifyShareholderData;
 
+  /**
+   * - Update a list of investors data
+   *
+   * Note that this procedure will fail if you're trying to revoke a shareholder's KYC
+   * Note that this procedure will fail if the security token symbol doesn't exist
+   * Note that this procedure will fail if General Transfer Manager isn't enabled
+   * Note that this procedure will fail if you're trying to modify nothing
```

Comment:
... if there is no difference between the "new" data and the data already present in the contract